packages/ResizeObserver/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@paprika/resize-observer",
```

Comment:
I wouldn't reuse an existing well-known name here. `ResizeObserver` API exists in modern browsers _and_ it has no visual user-facing component. `Resizer` works well enough?

Reply:
Okay, fair point.  I already use `<Resizer>` for the Storybook helper, and that name suggests that the component _does_ something besides just reporting... I also considered `ResizeListener` and `ResizeSensor`.  Any other suggetions?

Reply:
`ResizeDetector` sounds good to me 👍